include/swift/Basic/LangOptions.h

```diff
@@ -104,6 +104,10 @@ namespace swift {
     /// Enable features useful for running in the debugger.
     bool DebuggerSupport = false;
 
+    /// Enable the MemortBufferSerializerdModileImporter.
```

Comment:
Nit: typo: `Memory`

Reply:
And "Modile", the dangerous intersection between Module and Crocodile.